source/operations/concepts/architecture.rst

```diff
@@ -0,0 +1,189 @@
+.. _minio-architecture:
+
+=======================
+Deployment Architecture
+=======================
+
+.. default-domain:: minio
+
+.. contents:: Table of Contents
+   :local:
+   :depth: 2
+
+.. meta::
+   :keywords: topology, architecture, deployment, production
+   :description: Information on MinIO Deployment architecture and topology in production environments
+
+This page provides an overview of MinIO deployment architectures from a production perspective.
+For information on specific hardware or software configurations, see:
+
+- :ref:`Hardware checklists <minio-hardware-checklist>`
+- :ref:`Security Checklist <minio-security-checklist>`
+- :ref:`Software Checklist <minio-software-checklists>`
+- :ref:`Thresholds and Limits <minio-server-limits>`
+
+Distributed MinIO Deployments
+-----------------------------
+
+A production MinIO deployment consists of at least 4 MinIO hosts with homogeneous storage and compute resources.
+   MinIO aggregates these resources together as a :ref:`pool <minio-intro-server-pool>` and presents itself as a single object storage service.
+
+   .. figure:: /images/architecture/architecture-4-node-deploy.svg
+      :figwidth: 100%
+      :alt: 4 Node MinIO deployment with homogeneous storage and compute resources
+      :align: center
+
+      Each MinIO host in this pool has matching compute, storage, and network configurations
+
+MinIO provides best performance when using direct-attached storage (DAS), such as NVMe or SSD drives attached to a PCI-E controller board on the host machine. 
+   Storage controllers should present XFS-formatted drives in "Just a Bunch of Drives" (JBOD) configurations with no RAID, pooling, or other hardware/software resiliency layers.
+   Caching layers, either at the drive or the controller layer, also typically lead to unpredictable performance as I/O spikes and falls as the cache fills and clears. 
```

Comment:
An idea to break up this long sentence:
> MinIO recommends against caching, either at the drive or the controller layer. It causes I/O spikes as the cache fills and clears, resulting in unpredictable performance.

Reply:
Wait, do what now?
👀

Reply:
There's a little icon in the github icon that looks like a file with a `+` sign on it that auto-creates the template too, but it's a cute way of making requested suggestions.

Reply:
I was wondering how that worked, hadn't had time to look into it. Awesome!!